Blockzilla/UIConstants.swift

```diff
@@ -431,5 +431,12 @@ struct UIConstants {
         static let siriEraseTipDescription = "Add Siri shortcut"
         static let shareTrackersTipTitle = "%@ trackers blocked so far"
         static let encodingNameUTF8 = "utf-8"
+        static let googleAmpURLPrefix = "https://www.google.com/amp/s/"
     }
 }
+
+public enum KVOConstants: String {
```

Comment:
Since this only exists inside WebViewControler I would move this into it. A better name might be something like `KVOKeyPath`